library/objective-c/EnvoyEngine.mm

```diff
@@ -23,45 +164,92 @@ + (EnvoyStatus)runWithConfig:(NSString *)config logLevel:(NSString *)logLevel {
   }
 }
 
-+ (EnvoyStream)startStreamWithObserver:(EnvoyObserver *)observer {
-  NSLog(@"%@ not implemented, returning failure", NSStringFromSelector((SEL) __func__));
-  EnvoyStream stream;
-  stream.status = EnvoyStatusFailure;
-  return stream;
++ (void)setupEnvoy {
+  setup_envoy();
+}
+
+// TODO: move to Envoy
++ (EnvoyStream *)startHttpStreamForRequest:(EnvoyRequest *)request handler:(EnvoyObserver *)handler {
+  EnvoyStream *stream = [[EnvoyStream alloc] initWithEngine:engine,
+                                                    request:request,
+                                                   observer:handler];
 }
 
-+ (EnvoyStatus)sendHeaders:(EnvoyHeaders *)headers to:(EnvoyStream *)stream close:(BOOL)close {
-  NSLog(@"%@ not implemented, returning failure", NSStringFromSelector((SEL) __func__));
-  return EnvoyStatusFailure;
+@end
+
+@implementation EnvoyHttpStream
```

Comment:
I think you're missing the `EnvoyHttpStream` from this PR

Reply:
We are - we're still moving things around a little bit.